docs/zh-CN/docs/admin-manual/config/be-config.md

```diff
@@ -203,7 +203,7 @@ BE 重启后该配置将失效。如果想持久化修改结果，使用如下
 
 * 类型：int32
 * 描述：配置BE的所属于的集群id。
-  - 该值通常由FE通过心跳向BE下发，不需要额外进行配置。当确认某BE属于某一个确定的Drois集群时，可以进行配置，同时需要修改数据目录下的cluster_id文件，使二者相同。
+  - 该值通常由FE通过心跳向BE下发，不需要额外进行配置。当确认某BE属于某一个确定的Doris集群时，可以进行配置，同时需要修改数据目录下的cluster_id文件，使二者相同。
```

Comment:
Our usual norm is to add a space between Chinese and English

Reply:
fixed